src/Infrastructure/Share/Bus/CommandBus.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Share\Bus;
+
+use Symfony\Component\Messenger\Exception\HandlerFailedException;
+use Symfony\Component\Messenger\MessageBusInterface;
+use Throwable;
+
+final class CommandBus
+{
+    use MessageBusExceptionTrait;
+
+    /**
+     * @var MessageBusInterface
+     */
```

Comment:
Could this be in one line? 

```
/** @var MessageBusInterface */

Reply:
Don't worry about this, that's should be fixes in all places by #135.